src/components/productsList/ProductsList.js

```diff
@@ -0,0 +1,109 @@
+const normalizeClassName = classSelector => classSelector.replace('.', '');
+
+/**
+ * @module ProductsList
+ */
+class ProductsList {
+  /**
+   * @param {HTMLElement} element - The HTMLElement this module is constructed upon
+   * @param {Object} options - ConditionerJS's merged options
+   */
+  constructor(element, options) {
+    this._element = element
+    this._options = Object.assign(ProductsList.options, options)
+
+    this.load();
+  }
+
+  /**
+   * Base options
+   */
+  static options = {
+    navSelector: '.products-list-nav',
+    navTitleSelector: '.products-list-nav__title',
+    navLinksSelector: '.products-list-nav__link',
+    navLinksActiveSelector: '.products-list-nav__link--active',
+    contentItemsSelector: '.products-list-content__item',
+    activeLink: 1
+  }
+
+  _addEventListeners() {
+    const { navTitle, navLinks } = this._elements;
+
+    navTitle.addEventListener('click', this._toggleNavigation.bind(this));
+    navLinks.map(item => item.addEventListener('click', this._onSelectedProduct.bind(this)));
+  }
+
+  _toggleNavigation() {
+    const { nav } = this._elements;
+    const { navSelector } = this._options;
+
+    nav.classList.toggle(`${normalizeClassName(navSelector)}--opened`);
+  }
+
+  _onSelectedProduct(evt) {
+    evt.preventDefault();
+    const selectedProduct = evt.target;
+    const selectedProductId = selectedProduct.getAttribute('data-nav-item-id');
+
+    this._setActiveLink(selectedProductId);
+    this._setNavActiveLink();
+    this._setNavTitle();
+    this._toggleContent();
+    this._toggleNavigation();
+  }
+
+  _setNavActiveLink() {
+    const { navLinksSelector, activeLink } = this._options;
+    const navLinkActiveEl = this._element.querySelector(`${navLinksSelector}--active`);
+    const activeClassSelector = `${normalizeClassName(navLinksSelector)}--active`;
+    const navItemSelected = this._element.querySelector(`[data-nav-item-id="${activeLink}"]`);
```

Comment:
I would like to see these attributes in a separate constants object. This ensures that you can easily alter them later if you need to.